packages/std/src/math/uint128.rs

```diff
@@ -230,6 +232,8 @@ impl Uint128 {
     }
 }
 
+impl_mul_fraction!(Uint128, Uint256);
```

Comment:
I wonder if there's a macro trick in parsing out the struct name and doubling the integer part of it 🤔

Reply:
@webmaster128 if these updates look good to you, I can add the macro for the remaining Uint structs and write tests.

Reply:
Go for it!